src/Microsoft.TryDotNet.SimulatorGenerator/Microsoft.TryDotNet.SimulatorGenerator.csproj

```diff
@@ -12,8 +12,8 @@
     <ProjectReference Include="..\Microsoft.TryDotNet\Microsoft.TryDotNet.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="7.0.0" />
-    <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.22272.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="7.0.5" />
+    <PackageReference Include="System.CommandLine" Version="2.0.0-beta4.23113.1" />
```

Comment:
This is really a pre-beta5 and isn't compatible with the version being used in .NET Interactive. Not sure that will matter here, but just FYI.

Reply:
will use the same